petastorm/tests/test_end_to_end.py

```diff
@@ -857,3 +862,32 @@ def test_make_batch_reader_with_url_list(scalar_dataset):
             row_count += len(batch.id)
 
         assert row_count == 100
+
+
+def test_pyarrow_filters_make_reader(synthetic_dataset):
+    with make_reader(synthetic_dataset.url, workers_count=5, num_epochs=1,
+                     filters=[('partition_key', '=', 'p_5'), ]) as reader:
+        uv = set()
+        for data in reader:
+            uv.add(data[0])
+
+        assert uv == {'p_5'}
+
+
+def test_pyarrow_filters_make_batch_reader():
+    path = tempfile.mkdtemp()
+    url = 'file://' + path
+
+    data = create_test_scalar_dataset(url, 100, partition_by=['datetime'], two_datetime_vals=True)
+    SyntheticDataset = namedtuple('synthetic_dataset', ['url', 'data', 'path'])
```

Comment:
Not sure, why do we need an additional named tuple here? Can we just use `make_batch_reader(data.url, ...)`?

Reply:
Good point, fixed.